src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AnnotatedVariantProducer.java

```diff
@@ -29,51 +29,49 @@
     /**
      * Produces a VC from a {@link NovelAdjacencyReferenceLocations}
      * (consensus among different assemblies if they all point to the same breakpoint).
-     * @param novelAdjacencyReferenceLocations  consensus among different assemblies if they all point to the same breakpoint
+     * @param refLoc                            POS field of the VC
```

Comment:
`refLoc` is a `SimpleInterval` so I'd make it clear how it maps to the POS field